examples/panel/weights-and-biases-llm/README.md

```diff
@@ -0,0 +1,76 @@
+# Connect your Panel app to Weights and Biases
+
+This example demonstrates how to connect your Panel app to Weights and Biases. We will use the [Weights and Biases](https://www.wandb.com/) API to track several metrics when making prompts against a GPT model from OpenAI. The final application enables you to monitor token usage, prompt length, and the number of tokens generated during a user session of the application running while it is deployed. 
+
+## Account setup
+
+### Weights and Biases
+
+To run this example, you will need to create an account in Weights and Biases. You can do this by visiting the [Weights and Biases website sign up page - it's free](https://wandb.ai/site/). You will need to generate an API key, which you can obtain after you log in by visiting [https://wandb.ai/authorize](https://wandb.ai/authorize). 
+
+Once you log in, you can create a new project by visiting your user profile, selecting the project tab, and clicking on the "New Project" button. 
+
+![](./images/new_project.png)
+
+Create a new `.env` file in the root of the project and add the following lines:
```

Comment:
Do we need the `.env` file when deploying directly in web app? We can also linke this document for CLI deployments: https://docs.cloud.ploomber.io/en/latest/user-guide/cli.html#defining-environment-variables

Reply:
good point. we only need the .env when deploying from the CLI, but here the instructions are using the UI, all the environment variables should be defined as such: https://docs.cloud.ploomber.io/en/latest/user-guide/env-vars.html

Reply:
removed env references 

https://github.com/ploomber/doc/pull/113/commits/1331ff8941bc7ecaa15d34618d7f40b0a4bc8ad1

https://github.com/ploomber/doc/pull/113/commits/edf4e9136a64843ca582325618bab2a1c65687a2